### PR TITLE
V2Wizard: fix remove partition funtionallity (HMS-3753)

### PR DESCRIPTION
**Pull request by mgold1234**
ths commit fix the remove partition functionalli file system step

### CODE REVIEW

**Review by lavocatt — APPROVED** [review]
This works fine!

File: src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx [hunk 1]
```diff
@@ -163,6 +164,12 @@ const getSuffix = (mountpoint: string) => {
 
 const Row = ({ partition }: RowPropTypes) => {
   const [units, setUnits] = useState<Units>('MiB');
+  const dispatch = useAppDispatch();
+
+  // @ts-ignore
+  const handleRemovePartition = (id: uuidv4) => {
```
Comment (regexowl) [inline src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx:170]: Let's solve the problem so we don't have to silence it here. When you remove the `@ts-ignore` the message will say that a value is used as a type. `id` is initiated as a `uuidv4`, but the type of uuid is a string. So this should fix the problem :)
```suggestion
  const handleRemovePartition = (id: string) => {
```

**Review by regexowl — COMMENTED** [review]
Works great! Just one comment

File: src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx (hunk 1 above)
Comment (lavocatt) [reply src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx:170]: @regexowl what about preventing the usage of ts-ignore?

Comment (lavocatt) [reply src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx:170]: I mean with some rules for the linter.

Comment (regexowl) [reply src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx:170]: Thanks for the inspiration! :smile: https://github.com/osbuild/image-builder-frontend/pull/1799

**Review by regexowl — APPROVED** [review]
Works great, thank you! :sparkles:

**Pull request merged**